flutter/example/lib/main.dart

```diff
@@ -16,8 +16,6 @@ Future<void> main() async {
   await SentryFlutter.init(
     (options) {
       options.dsn = _exampleDsn;
-      // Change the 'sentry_flutter_example' below with your own package.
-      options.addInAppInclude('sentry_flutter_example');
```

Comment:
we could remove the `addInAppInclude` mention from the READMEs as well

Reply:
✅